src/Features/Core/Portable/AddDebuggerDisplay/AbstractAddDebuggerDisplayCodeRefactoringProvider.cs

```diff
@@ -115,7 +115,7 @@ private async Task<Document> ApplyAsync(Document document, TTypeDeclarationSynta
             var syntaxRoot = await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
             var semanticModel = await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
 
-            var editor = new SyntaxEditor(syntaxRoot, document.Project.Solution.Workspace.Services);
+            var editor = new SyntaxEditor(syntaxRoot, document.Project.Solution.Services);
```

Comment:
>SyntaxEditor [](http://example.com/codeflow?start=29&length=12)

Why not use document.GetSyntaxEditor here too?

Reply:
this code isn't shared in the CODE_STYLE layer.  so it wasn't necessary.   a lot of the changes were done mechanically.